cypress-shared/fixtures/multiProductPayload.json

```diff
@@ -0,0 +1,50 @@
+{
+  "data": {
```

Comment:
can u pls put this fixtures inside fedex-shipping of fixtures folder
So, that other will know that this is been used by fedex shipping app